### PR TITLE
Obaida albaroudi

### PR DESCRIPTION
**Pull request by Obaida-Albaroudi**

### CODE REVIEW

File: great-idea-website/css/index.css [hunk 1]
```diff
@@ -63,4 +63,93 @@ h1, h2, h3, h4, h5 {
...
+	width: 30%;
+}
+
+hr{
+	margin: 5%;
+}
+
+.end{
+	margin-left: 50%;
+}
```
Comment (AdetunjiShennaike) [inline great-idea-website/css/index.css:155]: try to add some comments for readability, like what would you like as a comment if you read someone else code for the first time, that would help you navigate and understand things more easily

File: great-idea-website/index.html [hunk 2]
```diff
@@ -14,50 +14,105 @@
...
-  <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+  </div>
+ 
+  <div>
+    <div class="left">
+      <h3>
+        Innovation<br>On</br>
+        Demand
+        <br>
+        <button>
```
Comment (AdetunjiShennaike) [inline great-idea-website/index.html:35]: nice use of break

File: great-idea-website/index.html [hunk 3]
```diff
@@ -14,50 +14,105 @@
...
+      123 Way 456 Street
+      Somewhere, USA
+      <br></br>
+      1 (888) 888-8888
+      <br></br>
+      sales@greatidea.io
+  </p>
+</div>
+    
+  <p class="end">
```
Comment (AdetunjiShennaike) [inline great-idea-website/index.html]: any particular reason for using h5?

**Review by AdetunjiShennaike — APPROVED** [review]

File: great-idea-website/css/index.css (hunk 1 above)
Comment (Obaida-Albaroudi) [reply great-idea-website/css/index.css:155]: Got it, will start commenting my code from now on!

File: great-idea-website/index.html (hunk 3 above)
Comment (Obaida-Albaroudi) [reply great-idea-website/index.html]: Honestly no real reason, I think I was just trying to align it with what I saw for the font of the text

**Pull request merged**